packages/thirdweb/src/react/web/utils/sortWallets.ts

```diff
@@ -38,7 +38,7 @@ export function sortWallets<T extends { id: string }>(
         }
         return 0;
       })
-      // show wallets with select ui first ( currently only in-app )
+      // show in-app wallets first
```

Comment:
Updated this comment given "show wallets with select ui first" isn't the case with ecosystem wallets